### ISSUE TITLE
Color support

### ISSUE THREAD

**Issue by JuraRusan** [Ajneb97/EasyCommandBlocker#9]
Request to add HEX colors, or at least Legacy (`&x&f&b&0&0&0&0`)

Thank you in advance!

**Comment by Ajneb97** [comment]
The plugin already has support for hex colors, use this format: #000000

**Comment by JuraRusan** [comment]
I tried using the format you suggested, but I don't get the color I expected.

```
blocked_command_default_actions:
  - "message: #f42f31 Error: #caeaf5You cannot execute the given command or the argument is invalid!"
```
![image](https://github.com/Ajneb97/EasyCommandBlocker/assets/76946109/24a7fa22-3c9d-48f7-9b7f-4fd17bb8eee7)

These colors were expected
![image](https://github.com/Ajneb97/EasyCommandBlocker/assets/76946109/c6dd6446-3e0d-48fe-a35b-ebb253bd72d4)
![image](https://github.com/Ajneb97/EasyCommandBlocker/assets/76946109/1806f28c-8e09-4bc5-b54c-9189f34de639)


Maybe some kind of error? Or am I doing it wrong?

**Comment by Ajneb97** [comment]
Are you using Spigot/Paper ?

**Comment by JuraRusan** [comment]
I use it on both a proxy (Waterfall) and a paper servers

```
This server is running Paper version git-Paper-84 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: c016e03)
You are 112 version(s) behind
Download the new version at: https://papermc.io/downloads/paper
Previous version: git-Paper-538 (MC: 1.19.4)
```

**Comment by JuraRusan** [comment]
Thanks a lot for the fix (1.8.2), everything works great!
